website/docs/r/waiting_room_event.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
we also need to update https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/website/cloudflare.erb to include the new resource in the sidebar.